server/channels/web/handlers.go

```diff
@@ -509,11 +509,6 @@ func (h *Handler) checkCSRFToken(c *Context, r *http.Request, token string, toke
 			sid := ""
 			userId := ""
 
-			if session != nil {
```

Comment:
So I see that that reaching this point implies that `session != nil` so the check was not needed, but we probably want to preserve the assigned values (or use them directly in the log fields declaration bellow)

Reply:
oh shoot good catch.